misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -271,8 +324,8 @@ public static void writeAPIDocs(PackageLibrary packageLib, String output, boolea
 
     private static void genProjectJson(PackageLibrary packageLib, String dataDir) {
 
-        File jsFile = new File(dataDir + File.separator + "doc_data.js");
-        File jsonFile = new File(dataDir + File.separator + "doc_data.json");
+        File jsFile = new File(dataDir + File.separator + DOC_DATA_JS);
+        File jsonFile = new File(dataDir + File.separator + DOC_DATA_JSON);
         if (jsFile.exists()) {
             boolean deleted = jsFile.delete();
             if (!deleted) {
```

Comment:
```suggestion
            if (! jsFile.delete()) {
```
Do we really need to have deleted variable?
We can change all the places in the class if it is okay to remove that variable.